frontend/survey-project/src/app/components/shared/ui/input/basic-input.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["basic-input.component.scss","basic-input.component.css"],"names":[],"mappings":"AAAA,0DAAA;AAEA;EACI,cAAA;ACAJ;;ADGE;EACE,sBAAA;EACA,cAAA;ACAJ;ADEI;EACE,kCAAA;EACA,iDAAA;ACAN;ADGI;EACE,kCAAA;ACDN;ADAI;EACE,kCAAA;ACDN;ADII;EACE,mBAAA;ACFN;;ADME;EACE,gBAAA;ACHJ;;ADME;EACE,sCAAA;ACHJ;;ADME;EACE,wCAAA;ACHJ;;ADME;EACE,sCAAA;ACHJ;;ADME;EACE,qCAAA;ACHJ;;ADME;EACE,mCAAA;ACHJ;;ADME;EACE,oCAAA;ACHJ;;ADME;EACE,oCAAA;ACHJ;;ADME;EACE,2CAAA;ACHJ","file":"basic-input.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.